napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -79,22 +77,6 @@ def _get_current_activity_dock(window: Window):
         callback=Window._toggle_play,
         keybindings=[{'primary': KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyP}],
     ),
-    Action(
```

Comment:
Unsure if it's safe to just remove an action. Is there a deprecation path to follow here?

Reply:
I would say that at least for actions related to experimental features we can just remove them without deprecation.